optimum/gptq/quantizer.py

```diff
@@ -635,8 +636,16 @@ def load_quantized_model(
         device_map = {"": torch.cuda.current_device()}
         logger.info("The device_map was not initialized." "Setting device_map to `{'':torch.cuda.current_device()}`.")
 
-    with open(os.path.join(save_folder, quant_config_name), "r", encoding="utf-8") as f:
-        quantize_config_dict = json.load(f)
+    # this branch is to add support for loading quantization_config from config.json
+    if os.path.exists(os.path.join(save_folder, CONFIG_NAME)):
```

Comment:
if the model is from transformers, you can access the config through `model.config`. So i suggest using the check `hasattr(model,"config")`. Same check for  `quantization_config`